src/main/java/edu/purdue/cs/percolator/VocareumFormatter.java

```diff
@@ -1,11 +1,94 @@
 package edu.purdue.cs.percolator;
 
 import com.github.tkutcher.jgrade.Grader;
+import com.github.tkutcher.jgrade.gradedtest.GradedTestResult;
 
+import java.io.BufferedWriter;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.List;
+
+/**
+ * Formats an output appropriate for Vocareum.
+ *
+ * @author kedarabhyankar
+ * @version 1.0
+ * @since 1.0
+ */
 class VocareumFormatter implements OutputFormatter {
 
+    /**
+     * The filename used by Vocareum for outputting grade information to.
+     */
+    public static final String OUT_FILE_NAME = "$vocareumGradeFile";
```

Comment:
Since there should be both a results file as well as an output file, I would rename this variable. Also, make it private.
```suggestion
    private final static String REPORT_FILE = "$vocareumReportFile";
```

Reply:
Also, I don't think that this will work when passing it to a FileWriter. I think you will need to use `System.getEnv()` to grab it first.